common/app/model/meta.scala

```diff
@@ -240,7 +277,7 @@ final case class MetaData(
     webUrl: String,
     section: Option[SectionId],
     pillar: Option[Pillar],
-    format: Option[ContentFormat],
+    format: ContentFormat,
```

Comment:
As [above](https://github.com/guardian/frontend/pull/23593/files#r591281844) with the decision to make this not an `Option`